.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: NullVoxPopuli/action-setup-pnpm@NullVoxPopuli-patch-1
```

Comment:
this branch doesn't exist anymore -- and is the reason for this PR.

I Just _Also_ moved to wyvox